ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -453,7 +454,7 @@ public func combineLatest<A, B, Error>(a: SignalProducer<A, Error>, b: SignalPro
 /// Combines the values of all the given producers, in the manner described by
 /// `combineLatestWith`.
 public func combineLatest<A, B, C, Error>(a: SignalProducer<A, Error>, b: SignalProducer<B, Error>, c: SignalProducer<C, Error>) -> SignalProducer<(A, B, C), Error> {
```

Comment:
Should these also be updated to make the parameters nameless? (As was done with the `Signal` versions of `combineLatest`)